lib/screens/app/app.dart

```diff
@@ -208,7 +208,7 @@ class _AppState extends State<App> with WidgetsBindingObserver {
                 : SizedBox.shrink()
             : SizedBox.shrink()
       ]),
-      title: isSelected
+      title: isSelected // TODO: Fix use of deprecated member
```

Comment:
I believe todo needs to follow TODO(githubuser) or the lint will fail

Reply:
cool

Reply:
The error there was actually not mine, I can just remove my TODO ;) I didn't introduce the use of title, that was already there

Reply:
Removed